README.md

```diff
@@ -1 +1,62 @@
-# -React-Capstone
\ No newline at end of file
+## Global Air Pollution Index
+
+In this project I have built <Global-Air-Pollution-Index>. by using the Latitude and the Longtude of any location in the worl we can see the severity of Air pollution. On the homepage I have included the top 10 countries where the Air Pollution is high.(You can check the ranking [here](https://worldpopulationreview.com/country-rankings/most-polluted-countries))
+
+
+## Built With
+
+- HTML, 
+- CSS and 
+- JavaScript/React(framework) and
+- [Air Pollution API](https://openweathermap.org/api/air-pollution)
+  
+## Live Demo
+
+[Live Demo](https://631b5bf9ac6b01187d804c8c--stalwart-hotteok-4f8c89.netlify.app/)
+
+## Getting Started
+
+To use this project you need to clone the repository to your local machine, and open the folder then run <npm-start> in the command line. To use npm start make sure react is installed in your system.
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+To work on this project locally you need to have Node.js installed in your machine.
+
+### Setup
+
+To setup this project first you should have to clone the repository and the go to the projects directory and run `npm install` command to install the needed dependency. After that run `npm start` command to run the project from local server to `http://localhost:8080/`.
+
+### Install
+
+You have to have Node.js installed before working on this project.
+
+### Authors
+
+👤 Eyasu Teshome
+- GitHub: [@EyasuTeshome](https://github.com/EyasuTeshome)
+- Twitter: [@EyasuTeshome](https://twitter.com/EyasuTeshome)
+- LinkedIn: [Eyasu Teshome](https://linkedin.com/in/EyasuTeshome)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- This project is part of week 2 of module 3 in the Microverse study program.
+- Thanks to the Microverse team for the great curriculum 🙌.
+- Thanks to Code Reviewers for the insightful feedback ⚡.
+- A great thanks to My coding partners, morning session team, and standup team Partners for their contributions 💘.
```

Comment:
- [ ] Please give appropriate credit to the author of the original design, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=The%20Creative%20Commons%20license%20of%20the%20design%20requires%20that%20you%20give%20appropriate%20credit%20to%20the%20author.%20Therefore%2C%20you%20must%20do%20it%20in%20the%20README%20of%20your%20project.) requirement.